### DISCUSSION TITLE
Problem with the sizes of thumbnails (generel understanding)

### DISCUSSION THREAD

**Discussion by ghost** [FriendsOfShopware/FroshPlatformThumbnailProcessor#72] [Q&A] [answered]
Hi

first of all a big thank you for all your work, whether here or in another PLugin. I love your works.

I have a problem understanding here, or do not get it set up correctly. After installing the two plugins (Bunny CDN and Thumbnail Processor), my thumbnails, and also generally linked images in the store are super small. What could be the reason for this? What information can I share to better isolate the problem? 

Thanks in advance and best regards
Mark Alsen
Moebelfaktor

Translated with www.DeepL.com/Translator (free version)

<img width="1469" alt="Screenshot 2022-12-05 193023" src="https://user-images.githubusercontent.com/105999018/205714457-48f148dd-c7a4-4648-8e10-c668ca682977.png">

**Comment by tinect** [comment, marked as answer]
Hi @markeymakey
it looks like your containers are missing some CSS for specifying width and or height.
Try setting to 100% or auto. Had this issue with some custom cms-elements, too, but had to consult our frontend dev :-(

EDIT: I just had a look into your shop.
![image](https://user-images.githubusercontent.com/135993/205725626-a8d63217-0047-4200-90f3-50c2d065a47c.png)
This is what you have in your own styles.

**Reply by ghost** [reply]
ahhh i see the plugin will add the class img.lazyloaded to all the images? Can one disable / enable that in the plugin somewhere? After using devtools to deactivate the class it looks just fine!

**Reply by tinect** [reply]
No, you can't because it's the basic function of this plugin in storefront :-(

I see you are using any other plugin for the images. While this is a paid-plugin, you might want to ask him, to try being compatible with this free plugin.
